webcompat/static/js/lib/qrcode.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var issues = issues || {};
+
+issues.QrView = Backbone.View.extend({
+  qrButton: null,
+  events: {
+    'click .wc-QrImage-launcher:not(.is-active)': 'showQr',
+    'click .wc-QrImage-launcher.is-active': 'closeQr'
+  },
+  keyboardEvents: {
+    'q': 'showQr'
+  },
+  render: function() {
+    // Extract page URL from bug description
+    var body = this.model.attributes.body;
```

Comment:
I think we probably want to use [`Model#get`](http://backbonejs.org/#Model-get) here. I don't think `attributes` is a public API. (I could be wrong though).